Userland/Utilities/tr.cpp

```diff
@@ -28,31 +175,26 @@ int main(int argc, char** argv)
         return 1;
     }
 
-    String from_complement;
-    StringView from_str;
+    String from_str = build_set(from_chars);
     if (complement_flag) {
-        auto original_set = StringView(from_chars);
         StringBuilder complement_set;
-        for (int i = 0; i < 256; i++) {
-            if (!original_set.contains(i))
-                complement_set.append(static_cast<int>(i));
+        for (int ch = 0; ch < 256; ch++) {
+            if (!from_str.find(static_cast<char>(ch)).has_value())
```

Comment:
Please use `String::contains(char)`, and if that doesn't exist, either implement it or use the original `StringView::contains(char)` method here and below.

it might also be worth sorting this `from_str` and using a binary search instead.

Reply:
Sure, sorting that string isn't a blocker for the PR, just a note.